dependencies.yaml

```
@@ -75,34 +78,33 @@ dependencies:
       - output_types: pyproject
         packages:
           - tomli ; python_version < '3.11'
-  cudatoolkit:
+  cuda_version:
     specific:
       - output_types: conda
         matrices:
-          - matrix:
-              cuda: "11.2"
-            packages:
-              - cuda-version=11.2
-              - cudatoolkit
           - matrix:
               cuda: "11.4"
             packages:
               - cuda-version=11.4
```

Looks like we lost CUDA 11.5 here.
```suggestion
          - matrix:
              cuda: "11.4"
            packages:
              - cuda-version=11.4
          - matrix:
              cuda: "11.5"
            packages:
              - cuda-version=11.5
```